nodejs/week3/WarmUp/app.js

```diff
@@ -0,0 +1,67 @@
+require("dotenv").config();
+const knex = require("knex")({
+  client: "mysql2",
+  connection: {
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    multipleStatements: true
+  }
+});
+let contactsTable;
+knex("contacts")
+  .columnInfo()
+  .then((result) => {
+    contactsTable = Object.keys(result);
+    console.log(
+      "\x1b[32m",
+      "app.js line:24 contactsTable",
```

Comment:
I am not sure if adding lines numbers in colsole log will always be accurate

Reply:
It's just a snippets for VSCode I am playing with. Theoretically, this particular snippet should automatically get the number of the line, where console.log is inserted... I will watch the snippet's behavior for a while, but for the moment I haven't noticed any errors in numbers.